### DISCUSSION TITLE
Request: Multiple lines on linechart with different spacing

### DISCUSSION THREAD

**Discussion by shesaid314** [Abhinandan-Kushwaha/react-native-gifted-charts#858] [Ideas]
Hello, I was wondering if it was possible to add different spacing on different lines for the LineChart. Adding a prop in the similar manner of data2, data3 but for spacing; spacing2.

**Comment by Abhinandan-Kushwaha** [comment]
Great suggestion! I will see the effort needed and let you know.

**Reply by shesaid314** [reply]
Thank you very much. Looking forward for your update.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @shesaid314 
This feature is now available from versions `1.4.44` onwards. 🎉

Please use the latest version of the library.

**Reply by shesaid314** [reply]
Awesome , thank you
